server/public/model/integration_action.go

```diff
@@ -335,6 +347,130 @@ func (r *OpenDialogRequest) DecodeAndVerifyTriggerId(s *ecdsa.PrivateKey, timeou
 	return DecodeAndVerifyTriggerId(r.TriggerId, s, timeout)
 }
 
+func (r *OpenDialogRequest) IsValid() error {
+	var multiErr *multierror.Error
+	if r.URL == "" || !IsValidHTTPURL(r.URL) {
+		multiErr = multierror.Append(multiErr, errors.New("invalid URL"))
+	}
```

Comment:
Plugins can use an absolute path instead of a URL, e.g. `/plugins/com.github.matterpoll.matterpoll/api/v1/polls/somepollid/delete/confirm` (https://github.com/matterpoll/matterpoll/blob/fc3e3e8fc5ec79a41bd7105533d53ec0450ac7d4/server/plugin/api.go#L710). The code should also allow that.